src/main/java/com/wavefront/sdk/common/ReconnectingSocket.java

```diff
@@ -149,7 +149,7 @@ public void write(String message) throws Exception {
       writeSuccesses.inc();
     } catch (Exception e) {
       try {
-        logger.log(Level.WARNING, "Attempting to reset socket connection.", e);
+        logger.log(Level.WARNING, "Attempting to reset socket connection.");
```

Comment:
I think we should change this message to something more user-friendly - something along the lines of "Unable to send data to [socket address], reconnecting" 
And, just to be able to still get it if needed, log the stacktrace as Level.FINE.